crates/cli/src/cli/mod.rs

```diff
@@ -223,22 +225,47 @@ impl StartSimnet {
             }
         }
 
-        for keypair_path in self.airdrop_keypair_path.iter() {
+        let mut airdrop_keypair_path = self.airdrop_keypair_path.clone();
+        if airdrop_keypair_path.is_empty() {
+            using_default_keypair = true;
+            airdrop_keypair_path.push(DEFAULT_SOLANA_KEYPAIR_PATH.clone());
+        }
+
+        let default_resolved_path = resolve_path(DEFAULT_SOLANA_KEYPAIR_PATH.as_str());
+
+        for keypair_path in airdrop_keypair_path.iter() {
             let path = resolve_path(keypair_path);
             match Keypair::read_from_file(&path) {
                 Ok(pubkey) => {
                     airdrop_addresses.push(pubkey.pubkey());
+                    if using_default_keypair && path == default_resolved_path {
+                        default_keypair_loaded = true;
+                    }
                 }
-                Err(e) => {
-                    errors.push(format!(
-                        "Unable to complete airdrop; Error reading keypair file: {}: {e}",
-                        path.display()
-                    ));
+                Err(_) => {
+                    match using_default_keypair {
+                        true => {
+                            errors.push(
+                                format!("No keypair found at {}, if you want to airdrop to a specific keypair provide the -k flag; skipping airdrops", DEFAULT_SOLANA_KEYPAIR_PATH.to_string())
+                            );
+                        }
+                        false => {
+                            errors.push(format!(
+                                "No keypair found at provided path {}; skipping airdrops;",
```

Comment:
If the user provides 3 keypairs and 2 of them are invalid, this will log:
```sh
No keypair found at provided path {}; skipping airdrops;
No keypair found at provided path {}; skipping airdrops;
Genesis airdrop for {} successful
```

It's weird we're saying "skipping" when we're not necessarily skipping